### PR TITLE
Make Maven Command Line Arguments for API Keys

### PR DESCRIPTION
**Pull request by nssafir**
To not have to manually enter each API key each time to test, the API keys are passed as command line arguments. Check the readme for more instructions about how to use it.

### CODE REVIEW

File: src/main/java/com/google/sps/data/GetAPIKeyUtil.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+public class GetAPIKeyUtil {
+
+  public String getAPIKey(String apiName) throws IOException {
+    System.out.println("in getAPIKey");
+    Properties properties = new Properties();
+    properties.load(GetAPIKeyUtil.class.getClassLoader().getResourceAsStream("project.properties"));  
+    return properties.getProperty(apiName + "Key");
+  }
+
+}
```
Comment (juanuribe28) [inline src/main/java/com/google/sps/data/GetAPIKeyUtil.java]: [Nit] Add newline.

File: src/main/java/com/google/sps/data/GetAPIKeyUtil.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+public class GetAPIKeyUtil {
+
+  public String getAPIKey(String apiName) throws IOException {
+    System.out.println("in getAPIKey");
+    Properties properties = new Properties();
+    properties.load(GetAPIKeyUtil.class.getClassLoader().getResourceAsStream("project.properties"));  
+    return properties.getProperty(apiName + "Key");
+  }
+
```
Comment (juanuribe28) [inline src/main/java/com/google/sps/data/GetAPIKeyUtil.java]: [Nit] Remove new line.

File: src/main/java/com/google/sps/data/GetAPIKeyUtil.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package com.google.sps.data;
+
+import java.io.InputStream;
+import java.util.Properties;
+import java.io.IOException;
+
+public class GetAPIKeyUtil {
+
+  public String getAPIKey(String apiName) throws IOException {
+    System.out.println("in getAPIKey");
```
Comment (juanuribe28) [inline src/main/java/com/google/sps/data/GetAPIKeyUtil.java]: Was this print statement for testing?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
 # STEP 2020 Capstone project
 
 Our objective is to provide users with relevant news and action-initiatives based on their interests We want users to be able to deeply understand topics related to their interests (as indicated by their social media activity) and have the resources to take action.
+
+# How To Run
+For security purposes, the API keys used in this applicaiton must be passed as command line arguments.
+To run, use the following command: 	`[mvn command] -DbooksKey=[Google Books API Key] -DprojectsKey=[Global Giving API Key] -DcivicsKey=[Google Civics API Key] -DlocationKey=[Google Geocoder API Key]`
```
Comment (juanuribe28) [inline README.md]: We could probably use the same API key for all the google apis.

File: src/main/resources/project.properties [hunk 5]
```diff
@@ -0,0 +1,4 @@
+booksKey = ${booksKey}
+projectsKey = ${projectsKey}
+civicsKey = ${civicsKey}
+locationKey = ${locationKey}
```
Comment (juanuribe28) [inline src/main/resources/project.properties]: [Nit] Add new line.

**Review by juanuribe28 — CHANGES_REQUESTED** [review]
This is really interesting.

File: src/main/java/com/google/sps/data/GetAPIKeyUtil.java (hunk 3 above)
Comment (nssafir) [reply src/main/java/com/google/sps/data/GetAPIKeyUtil.java]: Yes, forgot to take it out

File: README.md (hunk 4 above)
Comment (nssafir) [reply README.md]: I'm going to look into this

**Review by juanuribe28 — APPROVED** [review]

**Review by seangilpin — APPROVED** [review]
Very nice!

**Pull request merged**